Api/Src/Models/User.ts

```diff
@@ -0,0 +1,48 @@
+import * as mongoose from 'mongoose';
+
+export enum UserType {
+    Participant,
+    Mentor, 
+    Admin,
+};
+
+export interface UserModel {
```

Comment:
Model looks good ;) When I've created endpoints documentation I also created user model. Our models are similar, but in my model are additional fields like grades and status. You can check this in my pull request. Maybe we should add these fields into your model? We can talk about it during meeting. ;)

Reply:
Ok, I added a status to user model and changed the names in the model to be the same. I suggest add grades after creating or adding files about grades. In user model we must add grades model, and this is also not done yet.

Reply:
Ok, we will add grades to user when model will be complete.